src/examples/src/widgets/form/Basic.tsx

```diff
@@ -23,7 +23,10 @@ const App = factory(function({ middleware: { icache } }) {
 
 	return (
 		<virtual>
-			<Form onValue={(values) => icache.set('basic', { ...icache.get('basic'), ...values })}>
+			<Form
+				name="basicForm"
```

Comment:
Since this is mean to help with action forms, and more specifically nested forms, maybe we should either:

1. Add `name` to an action form example.
2. Create a new example for nested forms with the `name` property.

Or maybe both.

Reply:
i think both :)

Reply:
I added a name to a new action form example. I didn't add a nested form example since it's [not supported by the spec](https://html.spec.whatwg.org/multipage/forms.html#the-form-element)